bin/get_data.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+# Limit maximum threads to avoid potential issues
+export OMP_NUM_THREADS=4
+export MKL_NUM_THREADS=${OMP_NUM_THREADS}
+export NUMEXPR_NUM_THREADS=${OMP_NUM_THREADS}
+export OPENBLAS_NUM_THREADS=${OMP_NUM_THREADS}
+export VECLIB_MAXIMUM_THREADS=${OMP_NUM_THREADS}
+
+# Set path, filenames and variables used later in the script
+export WORK="$(dirname $0)/.."
+cd $WORK
+export DATE=$1
+export TIME=$2
+export BASE=${DATE}T${TIME}.an
+export GRIB=grib/${BASE}.grib
+export mlfile=mss/${BASE}.ml.nc
+export plfile=mss/${BASE}.pl.nc
+export alfile=mss/${BASE}.al.nc
+export tlfile=mss/${BASE}.tl.nc
+export pvfile=mss/${BASE}.pv.nc
+export sfcfile=mss/${BASE}.sfc.nc
+export tmpfile=mss/.${BASE}.tmp.nc
+export gph_levels=0,25,50,75,100,125,150,175,200,225,250,275,300,325,350,375,400,425,450,475,500,525,550,575,600,625,650,675,700,725,750,775,800,825,850,875,900,925,950,975,1000,1025,1050,1075,1100,1125,1150,1175,1200,1225,1250,1275,1300,1325,1350,1375,1400,1425,1450,1475,1500,1525,1550,1575,1600,1625,1650,1675,1700,1725,1750,1775,1800,1825,1850,1875,1900,1925,1950,1975,2000,2050,2100,2150,2200,2250,2300,2350,2400,2450,2500,2550,2600,2650,2700,2750,2800,2850,2900,2950,3000,3050,3100,3150,3200,3250,3300,3350,3400,3450,3500,3550,3600,3650,3700,3750,3800,3850,3900,3950,4000,4100,4200,4300,4400,4500,4600,4700,4800,4900,5000,5100,5200,5300,5400,5500,5600,5700,5800,5900,6000
+init_date=$(date +%Y-%m-%dT%H:%M:%S)
+if [[ "$init_date" > "$DATE" ]] 
+then 
+    init_date="${DATE}T${TIME}"
+fi
+export time_units="hours since ${init_date}"
+
+# Download ml, sfc, pv and pt files
+echo "Downloading files, this might take a long time!"
+if [ ! -f grib/${BASE}.ml.grib ]; then
+    echo test
+    python bin/download_an_all.py $DATE $TIME ml &
```

Comment:
Das ist jetzt doppelt paralellisiert. Mach das doch bitte in python *oder* bash.